README.md

```diff
@@ -144,13 +145,31 @@ You can add a new secret parameter [using AWS SSM](https://www.notion.so/Working
 
 ## Development Tools
 
+1. Set up a [virtual environment](https://docs.python.org/3/library/venv.html) (or let your IDE do it for you).
 We use [Poetry](https://python-poetry.org/) to manage dependencies. It also helps with pinning dependency and python
-versions.
+2. Run `pip install -r requirements.txt`
+versions. We also use [pre-commit](https://pre-commit.com/) with hooks for [isort](https://pycqa.github.io/isort/),
+3. Run `pip install pre-commit`
+[black](https://github.com/psf/black), and [flake8](https://flake8.pycqa.org/en/latest/) for consistent code style and
+readability. Note that this means code that doesn't meet the rules will fail to commit until it is fixed.
 
 ### Setup
 
 1. [Install Poetry](https://python-poetry.org/docs/#installation).
 2. Run `poetry install --no-root`
+3. Make sure the virtual environment is active, then
+4. Run `pre-commit install`
+4. Run `pre-commit install`
```

Comment:
Duplicate line here. Nothing crazy so I can push up a change

Reply:
Thanks Duy! If there's nothing else, feel free to approve it 😇